parlai/crowdsourcing/utils/tests.py

```diff
@@ -89,10 +89,10 @@ def _set_up_config(
             self.config = compose(
                 config_name="example",
                 overrides=[
-                    f'+mephisto.blueprint._blueprint_type={blueprint_type}',
-                    f'+mephisto.blueprint.link_task_source=False',
-                    f'+mephisto/architect=mock',
-                    f'+mephisto/provider=mock',
+                    f'mephisto.blueprint._blueprint_type={blueprint_type}',
```

Comment:
Ooof, fiddling with the pluses is painful -__- Hmm what does the double-plus mean? Does that allow for either adding *or* overwriting the param?